### PR TITLE
[CI] Abort image building process if the npcap file non found

### PR DESCRIPTION
**Pull request by pazone**
`1.79` release caused a problem: the `npcap-1.79.exe` was missing and broken images were published despite of that. 
Now it will stop building process if the required file is missing.

### CODE REVIEW

**Review by andrewkroh — APPROVED** [review]
LGTM. I think there is also value in https://github.com/elastic/golang-crossbuild/pull/383 which is fail the `docker build` if the file is not present.

**Review by v1v — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]

**Comment by v1v** [comment]
@pazone , this PR should be backported, too; I added the required label.

All the changes in `golang-crossbuild@main` should be backported to the branch matching the support Golang versions by default. There might be cases that are not required... but the golden-rule is backport it

Golang [supports two versions](https://go.dev/doc/devel/release#policy), that's the same principle for this particular repository.

In other words, as of today, Golang supports `1.21` and `1.20`, and that's the same in this repository, with the difference that `main` is using `1.21`.
